---
id: agm-006
title: "AGM-006 Nested AGENTS.md Hierarchy"
sidebar_label: "AGM-006"
---

## Summary

- **Rule ID**: `AGM-006`
- **Severity**: `MEDIUM`
- **Category**: `AGENTS.md`
- **Normative Level**: `BEST_PRACTICE`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `all`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://developers.openai.com/codex/guides/agents-md
- https://docs.cline.bot/features/custom-instructions
- https://github.com/github/docs/changelog/2025-06-17-github-copilot-coding-agent-now-supports-agents-md-custom-instructions

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
# Instructions
Do everything automatically.
```

### Valid

```markdown
## Project Instructions
- Use AGENTS.md as instruction entrypoint
- Keep commands explicit and test changes
```
