---
id: as-008
title: "AS-008 Description Too Short"
sidebar_label: "AS-008"
---

## Summary

- **Rule ID**: `AS-008`
- **Severity**: `HIGH`
- **Category**: `Agent Skills`
- **Normative Level**: `MUST`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `all`
- **Version Range**: `unspecified`
- **Spec Revision**: `1.0`

## Evidence Sources

- https://agentskills.io/specification

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
---
description: Deploys production changes
---

# deploy
Use the skill now.
```

### Valid

```markdown
---
name: deploy-prod
description: Deploy production with explicit checks
---

# deploy-prod
Run rollout checks before deployment.
```
