---
id: cop-004
title: "COP-004 Unknown Frontmatter Keys"
sidebar_label: "COP-004"
---

## Summary

- **Rule ID**: `COP-004`
- **Severity**: `MEDIUM`
- **Category**: `GitHub Copilot`
- **Normative Level**: `SHOULD`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `github-copilot`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://docs.github.com/en/copilot/customizing-copilot

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
# Copilot Instructions
Write whatever code seems fine.
```

### Valid

```markdown
# Copilot Instructions
Use project coding standards and keep tests updated.
```
