---
id: cop-006
title: "COP-006: File Length Limit - GitHub Copilot"
sidebar_label: "COP-006"
description: "agnix rule COP-006 checks for file length limit in github copilot files. Severity: MEDIUM. See examples and fix guidance."
keywords: ["COP-006", "file length limit", "github copilot", "validation", "agnix", "linter"]
---

## Summary

- **Rule ID**: `COP-006`
- **Severity**: `MEDIUM`
- **Category**: `GitHub Copilot`
- **Normative Level**: `SHOULD`
- **Auto-Fix**: `No`
- **Verified On**: `2026-02-07`

## Applicability

- **Tool**: `github-copilot`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://docs.github.com/en/copilot/customizing-copilot

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
# Copilot Instructions
Write whatever code seems fine.
```

### Valid

```markdown
# Copilot Instructions
Use project coding standards and keep tests updated.
```
