---
id: oc-003
title: "OC-003: opencode.json Parse Error - opencode"
sidebar_label: "OC-003"
description: "agnix rule OC-003 checks for opencode.json parse error in opencode files. Severity: HIGH. See examples and fix guidance."
keywords: ["OC-003", "opencode.json parse error", "opencode", "validation", "agnix", "linter"]
---

## Summary

- **Rule ID**: `OC-003`
- **Severity**: `HIGH`
- **Category**: `opencode`
- **Normative Level**: `MUST`
- **Auto-Fix**: `No`
- **Verified On**: `2026-02-07`

## Applicability

- **Tool**: `opencode`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://opencode.ai/docs/config

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```text
Configuration omitted required fields for this rule.```

### Valid

```text
Configuration includes required fields and follows the rule.```
