---
id: agm-001
title: "AGM-001 Valid Markdown Structure"
sidebar_label: "AGM-001"
---

## Summary

- **Rule ID**: `AGM-001`
- **Severity**: `HIGH`
- **Category**: `AGENTS.md`
- **Normative Level**: `MUST`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `all`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://developers.openai.com/codex/guides/agents-md
- https://docs.cursor.com/en/context
- https://docs.cline.bot/features/custom-instructions

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
# Instructions
Do everything automatically.
```

### Valid

```markdown
## Project Instructions
- Use AGENTS.md as instruction entrypoint
- Keep commands explicit and test changes
```
