---
id: xml-003
title: "XML-003 Unmatched Closing Tag"
sidebar_label: "XML-003"
---

## Summary

- **Rule ID**: `XML-003`
- **Severity**: `HIGH`
- **Category**: `XML`
- **Normative Level**: `MUST`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `all`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://www.w3.org/TR/xml/

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```xml
<analysis><rule id="XML-001"></analysis>
```

### Valid

```xml
<analysis><rule id="XML-001">ok</rule></analysis>
```
