---
id: xp-002
title: "XP-002 AGENTS.md Platform Compatibility"
sidebar_label: "XP-002"
---

## Summary

- **Rule ID**: `XP-002`
- **Severity**: `HIGH`
- **Category**: `Cross-Platform`
- **Normative Level**: `SHOULD`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `all`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://developers.openai.com/codex/guides/agents-md
- https://opencode.ai/docs/rules
- https://docs.cursor.com/en/context
- https://docs.cline.bot/features/custom-instructions
- https://github.com/github/docs/changelog/2025-06-17-github-copilot-coding-agent-now-supports-agents-md-custom-instructions

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
Use only CLAUDE.md instructions and ignore AGENTS.md.
```

### Valid

```markdown
Use both CLAUDE.md and AGENTS.md with explicit precedence and conflict handling.
```
