---
id: ref-001
title: "REF-001 Import File Not Found"
sidebar_label: "REF-001"
---

## Summary

- **Rule ID**: `REF-001`
- **Severity**: `HIGH`
- **Category**: `References`
- **Normative Level**: `MUST`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `claude-code`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://code.claude.com/docs/en/memory

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
[Spec](./missing-file.md)
```

### Valid

```markdown
[Spec](./VALIDATION-RULES.md)
```
