---
id: as-010
title: "AS-010 Missing Trigger Phrase"
sidebar_label: "AS-010"
---

## Summary

- **Rule ID**: `AS-010`
- **Severity**: `MEDIUM`
- **Category**: `Agent Skills`
- **Normative Level**: `SHOULD`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `claude-code`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://github.com/anthropics/awesome-slash
- https://platform.claude.com/docs

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
---
description: Deploys production changes
---

# deploy
Use the skill now.
```

### Valid

```markdown
---
name: deploy-prod
description: Deploy production with explicit checks
---

# deploy-prod
Run rollout checks before deployment.
```
