---
id: cur-004
title: "CUR-004 Invalid Glob Pattern in globs Field"
sidebar_label: "CUR-004"
---

## Summary

- **Rule ID**: `CUR-004`
- **Severity**: `HIGH`
- **Category**: `Cursor`
- **Normative Level**: `MUST`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `cursor`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://docs.cursor.com/en/context

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
# Rule
No metadata block
```

### Valid

```markdown
---
description: Cursor rule for repository policy
---
Use project-specific guidance.
```
