---
id: pe-004
title: "PE-004 Ambiguous Instructions"
sidebar_label: "PE-004"
---

## Summary

- **Rule ID**: `PE-004`
- **Severity**: `MEDIUM`
- **Category**: `Prompt Engineering`
- **Normative Level**: `SHOULD`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `all`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://docs.anthropic.com/en/docs/build-with-claude/prompt-engineering/overview

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
Do the task quickly.
```

### Valid

```markdown
## Objective
Validate AGENTS.md files for schema and policy compliance.

## Output Format
Return JSON diagnostics grouped by file.
```
