---
id: ver-001
title: "VER-001 No Tool/Spec Versions Pinned"
sidebar_label: "VER-001"
---

## Summary

- **Rule ID**: `VER-001`
- **Severity**: `LOW`
- **Category**: `Version Awareness`
- **Normative Level**: `BEST_PRACTICE`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `all`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://github.com/avifenesh/agnix/issues/104

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
Pin MCP schema to an outdated version without fallback behavior.
```

### Valid

```markdown
Declare supported version range and degrade gracefully outside the range.
```
