---
id: xp-004
title: "XP-004 Conflicting Build/Test Commands"
sidebar_label: "XP-004"
---

## Summary

- **Rule ID**: `XP-004`
- **Severity**: `MEDIUM`
- **Category**: `Cross-Platform`
- **Normative Level**: `SHOULD`
- **Verified On**: `2026-02-04`

## Applicability

- **Tool**: `all`
- **Version Range**: `unspecified`
- **Spec Revision**: `unspecified`

## Evidence Sources

- https://developers.openai.com/codex/guides/agents-md
- https://docs.cursor.com/en/context

## Test Coverage Metadata

- Unit tests: `true`
- Fixture tests: `true`
- E2E tests: `false`

## Examples

The following examples are illustrative snippets for this rule category.

### Invalid

```markdown
Use only CLAUDE.md instructions and ignore AGENTS.md.
```

### Valid

```markdown
Use both CLAUDE.md and AGENTS.md with explicit precedence and conflict handling.
```
